index.js

```diff
@@ -86,6 +86,9 @@ function togpx( geojson, options ) {
           "name": options.featureTitle(f.properties),
           "desc": options.featureDescription(f.properties)
         };
+        if (coordinates[2]) {
```

Comment:
This doesn't work when the z-coordinate is exactly 0 (or something else evaluating to false). Better to check against `!== undefined` or replace it with `coordinates.length > 2`.